src/vs/base/browser/ui/list/listView.ts

```diff
@@ -566,7 +568,12 @@ export class ListView<T> implements ISpliceable<T>, IDisposable {
 
 		if (!item.row) {
 			item.row = this.cache.alloc(item.templateId);
-			item.row!.domNode!.setAttribute('role', 'treeitem');
+			const role = this.ariaProvider.getRole ? this.ariaProvider.getRole(item.element) : 'treeitem';
+			item.row!.domNode!.setAttribute('role', role);
+			const checked = this.ariaProvider.isChecked ? this.ariaProvider.isChecked(item.element) : undefined;
+			if (checked !== undefined) {
```

Comment:
`typeof checked !== 'undefined'`

Reply:
Ack.
I was not aware of the difference thus I found this https://stackoverflow.com/questions/4725603/variable-undefined-vs-typeof-variable-undefined